homeassistant/components/guardian/button.py

```diff
@@ -56,15 +56,15 @@ async def _async_valve_reset(client: Client) -> None:
 BUTTON_DESCRIPTIONS = (
     ValveControllerButtonDescription(
         key=BUTTON_KIND_REBOOT,
-        name="Reboot",
```

Comment:
Do we not need a translation key here (in place of this removed name)?

Reply:
Nope, it will fall back to the translation provided by the device class.
Resulting in less translation keys, and thus less to translate for the translation community.